en/docs/develop/hosting-authentication-endpoint-on-a-different-server.md

```diff
@@ -83,28 +91,48 @@ Server).
     - cxf-rt-rs-extension-search-*.jar 
     - cxf-rt-rs-service-description-*.jar 
     - cxf-rt-transports-http-*.jar 
-    - org.wso2.carbon.bootstrap-*.jar 
-    - tomcat-juli-*.jar 
-    - xercesImpl-*.jar 
-    - geronimo-jta_*.jar 
-    - stax2-api-*.jar 
-    - woodstox-core-asl-*.jar 
-    - log4j-*.jar 
-    - pax-logging-api-*.jar 
     ```
+    
+3.  Copy the following .jar files from the `<IS_HOME>/lib/` directory to the `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/lib` directory. 
+    
+    ```xml
+    - xercesImpl-*.jar
+    - geronimo-jta_*.jar
+    - stax2-api-*.jar
+    - woodstox-core-asl-*.jar
+    ```
+    
+4.  Copy the following .jar files from the `<IS_HOME>/bin/` directory to the `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/lib` directory. 
+    
+    ```xml
+    - org.wso2.carbon.bootstrap-*.jar
+    - tomcat-juli-*.jar
+    ```
+
+5. Copy the following .jar files from the `<IS_HOME>/repository/components/tools/forget-me/lib/` directory to the `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/lib` directory. 
 
-2.  Copy the following .jar files from the `IS_HOME>/lib/runtimes/cxf/` directory to the `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/lib`
-    directory.  
+    ```xml
+    - log4j-*.jar
+    - pax-logging-api-*.jar
+    ```
+6. Copy the .jar files from `<IS_HOME>repository/components/features/org.wso2.carbon.identity.application.authentication.framework.server_*/runtimes/cxf3/org.wso2.carbon.identity.application.authentication.endpoint.util-*.jar` to the`<WEBAPP_HOME>/authenticationendpoint/WEB-INF/lib` directory. 
     
+7. Copy the `RecoveryEndpointConfig.properties` file from the `<IS-HOME>/repository/conf/identity` directory to the `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/classes` directory.
+
+8. Navigate to `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/classes/RecoveryEndpointConfig.properties` file and uncomment the following line.
     ```xml
-    - javax.ws.rs-
-    - api `-2.0-m10.jar
-    - cxf `-bundle-2.7.16.wso2v1.jar
-    - neethi `-3.0.3.jar
-    - wsdl4j-1.6.3.jar
+    identity.server.service.contextURL=https://localhost:9443
+    ```
+9. Navigate to `<WEBAPP_HOME>/authenticationendpoint/WEB-INF/classes/EndpointConfig.properties` file and change the following line,
+    ```
```

Comment:
Don't need this code block here if we are using the above line.

Reply:
fixed in [8df37a9](https://github.com/wso2/docs-is/pull/2853/commits/8df37a9d2e6fee032cd30c4fad51df1d884d0bc3)